src/agent/include/agent.hpp

```diff
@@ -5,6 +5,7 @@
 #include <command_handler.hpp>
 #include <communicator.hpp>
 #include <configuration_parser.hpp>
+#include <icommand_store.hpp>
```

Comment:
To remove this, let's create an interface `icommand_handler.hpp` as part of https://github.com/wazuh/wazuh-agent/issues/593. This is a similar solution to the one implemented with the queue (`imultiutype_queue.hpp`).
cc @Nicogp

Reply:
It has been added as a task in #593